Pipfile

```diff
@@ -38,10 +38,19 @@ scipy = "==1.6.2"
 seaborn = "*"
 sqlalchemy = "*"
 statsmodels = "*"
-tensorboard = "==1.15.0"
+tensorboard = "==2.6"
```

Comment:
If we change the tensorboard version then the metadata of the image should be updated to reflect it in the spawner